dev_requirements.txt

```diff
@@ -5,4 +5,4 @@ pytest>=7.0.0
 setuptools
 tox==3.24.4
```

Comment:
You can probably drop tox from here and all of the other linters too.
Did you run the manually btw?
In general I think it would be a good idea to add a workflow to run the linters. But this can be done in a different PR.